service_mesh/service_mesh_install/updating-ossm.adoc

```diff
@@ -4,7 +4,21 @@ include::modules/ossm-document-attributes.adoc[]
 :context: installing-ossm
 toc::[]
 
-If you selected the automatic update stream, updating {ProductName} doesn't require any extra steps. 
+If you're updating from {ProductName} 1.0 to 1.1, you must update the `ServiceMeshControlPlane` resource YAML file to enable the update. 
```

Comment:
I'd remove "YAML file", since it may confuse users - all they need to do is modify the resource (whether they do this via `oc edit`, the web console editor, or by actually editing the YAML file and re-applying it, is a different story).

Will this sentence as a whole be clear to readers? It doesn't clearly say that updating only updates the operator. It seems to say that when you update, nothing actually gets updated until you also modify the resource. Maybe change the ending to something like this: "... to update the control plane components to the new version."